laurent22/joplin
# Desktop: Fixes #10668: Tags and Delete note not being available on Search and on All Notes list

Fixes https://github.com/laurent22/joplin/issues/10668

# Description

The issue existed because the state `selectedFolderId` doesn't change when we select All Notes or do a Search. Since we use the `selectedFolderId` to check if it is identical to the static trash id to identify if we are in the "Trash context", switching from Trash to all notes was like we were still in the same context.

# Testing

### Test case 1

1. Go to Trash
2. Go to All notes
3. Right-click on a note in Note list : *Delete note* and *Tags* action should be available and not greyed out

### Test case 2

1. Go to Trash
2. Go to All notes
3. Make a Search for some note that exists
3. Right-click on a note in Note list : *Delete note* and *Tags* actions should be available and not greyed out
